cfgov/unprocessed/js/modules/youtube-api.js

```diff
@@ -0,0 +1,86 @@
+/* ==========================================================================
+   YouTube API
+   ========================================================================== */
+
+import jsLoader from './util/js-loader';
+
+const IMAGE_URL = 'https://img.youtube.com/vi/%video_id%/maxresdefault.jpg';
+const SCRIPT_API = 'https://www.youtube.com/iframe_api';
+
+/**
+ * The "onYouTubeIframeAPIReady" function is automatically called when the
+ * YouTube iFrame API is done loading.
```

Comment:
This is trivial, but YouTube calls it the "IFrame API" (capital I). If you care, might be nice to use that throughout here. (And also ensure it's "YouTube" with a capital 'T' everywhere.)

Reply:
Fixed